freetextresponse/freetextresponse.py

```diff
@@ -33,8 +33,13 @@ def workbench_scenarios():
         scenarios = [
             ('Free-text Response XBlock',
              '''<sequence_demo>
-                    <freetextresponse />
```

Comment:
I'm not seeing that line here. We should always provide an example of the default case, with no customization.